ui/pages/onboarding-flow/creation-successful/creation-successful.js

```diff
@@ -32,84 +43,161 @@ export default function CreationSuccessful() {
   const t = useI18nContext();
   const trackEvent = useContext(MetaMetricsContext);
   const firstTimeFlowType = useSelector(getFirstTimeFlowType);
+  const seedPhraseBackedUp = useSelector(getSeedPhraseBackedUp);
+  const learnMoreLink =
+    'https://support.metamask.io/hc/en-us/articles/360015489591-Basic-Safety-and-Security-Tips-for-MetaMask';
+  const learnHowToKeepWordsSafe =
+    'https://community.metamask.io/t/what-is-a-secret-recovery-phrase-and-how-to-keep-your-crypto-wallet-secure/3440';
 
   const { createSession } = useCreateSession();
 
   const isProfileSyncingEnabled = useSelector(selectIsProfileSyncingEnabled);
   const participateInMetaMetrics = useSelector(selectParticipateInMetaMetrics);
 
   return (
-    <div className="creation-successful" data-testid="creation-successful">
-      <Box textAlign={TEXT_ALIGN.CENTER}>
-        <img src="./images/tada.png" />
-        <Typography
-          variant={TypographyVariant.H2}
-          fontWeight={FONT_WEIGHT.BOLD}
+    <Box
+      className="creation-successful"
+      data-testid="creation-successful"
+      display={Display.Flex}
+      flexDirection={FlexDirection.Column}
+    >
+      <Box
+        display={Display.Flex}
+        flexDirection={FlexDirection.Column}
+        justifyContent={JustifyContent.center}
+        marginTop={6}
+      >
+        <Text
+          justifyContent={JustifyContent.center}
+          marginBottom={4}
+          style={{
+            alignSelf: AlignItems.center,
+            fontSize: '70px',
+          }}
+        >
+          {firstTimeFlowType === FirstTimeFlowType.create &&
+          !seedPhraseBackedUp ? (
+            <span>🔓</span>
```

Comment:
Mega-nit: The ternary could be inside the `<span>`s to avoid using the same wrapper twice.

Reply:
True!